docs/user_guides/CASE_MANAGERS.md

```diff
@@ -0,0 +1,74 @@
+# Using DARIA as a Case Manager
+
+The main point of DARIA is to ensure case managers can focus on conversation with patients, rather than fighting with computers. DARIA tries hard to make sure case managers don't need to sweat the small stuff, and to organize patient information consistently.
+
+Case managers will spend most of their time on the `Call List View` and the `Patient Record View`. This document will go through those views and what they're used for.
+
+## The call list view
+
+When you log in and pick your line, and see a header of `Build your call list`, you are on the `Call List View`. This is where you can plan your work for the evening, make a list of people to call, log any new callers, and keep tabs on who you've called.
+
+### Build your call list
+
+You use this utility to populate your call list for the evening. Entering either a phone number, part of a phone number, or a name will search patients for a match and return any matches it finds that aren't already in your call list. You can add people to your call list by clicking the `Add` link after searching for them.
```

Comment:
Should we say "populate your call list for the day" instead of "evening" since people could be doing this any time? We say "day" later

Reply:
good call!